client/components/social-buttons/google.js

```diff
@@ -92,7 +93,7 @@ class GoogleLoginButton extends Component {
 		if ( this.state.error ) {
 			this.setState( {
 				showError: ! this.state.showError,
-				errorRef: event.target,
+				errorRef: closest( event.target, '.social-buttons__button', true ),
```

Comment:
@pento https://developer.mozilla.org/en/docs/Web/API/Event/currentTarget, `event.currentTarget` will do the trick :)